appserver/tests/payara-samples/samples/realm-identity-stores/src/test/java/fish/payara/samples/realm/identity/store/ExistingRealmIdentityStoreDefinitionTest.java

```diff
@@ -89,7 +87,6 @@ public static WebArchive createDeployment() {
                         PersonController.class,
                         AuthoritiesConstants.class
                 )
-                .addAsWebInfResource(new File(WEBAPP_SOURCE, "WEB-INF/glassfish-web.xml"))
                 .addAsWebInfResource(EmptyAsset.INSTANCE, ArchivePaths.create("beans.xml"));
```

Comment:
Even empty beans.xml is superflous in my opinion

Reply:
No classes added to test war with CDI scoped annotation.